itl/command-plugins.conf

```diff
@@ -512,6 +521,10 @@ object CheckCommand "http" {
 			set_if = "$http_extendedperfdata$"
 			description = "Print additional perfdata"
 		}
+		"-U" = {
+			value = "$http_showurl$"
+			description = "Print URL in msg output in plain text"
+		}
```

Comment:
Where did you get that option from? As far as I can see, there is no such option listed in the [plugin documentation](https://www.monitoring-plugins.org/doc/man/check_http.html).